docker/web/public/week6/createqrcard.php

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
All these are my new files are a dead give away that you're committing to the wrong branch. You shouldn't be adding other people files.